base/OpenGLVisitor.cpp

```diff
@@ -155,6 +172,34 @@ void OpenGLVisitor::visit(Group* g) {
 #endif
             vector<int>::iterator indexIter = (*polyIter)->m_vIndex.begin();
             vector<int>::iterator indexEnd = (*polyIter)->m_vIndex.end();
+            #ifdef HAVE_OPENGLES
+            GLuint vbo;
+            glGenBuffers(1, &vbo);
+            glBindBuffer(GL_ARRAY_BUFFER, vbo);
+            float *verts = (float *)calloc((*polyIter)->m_vIndex.size(), 5 * sizeof(float));
```

Comment:
An other option would be to use display lists ?

Reply:
Don't hesitate to update it if you are planning a refactoring.